docs/UserGuide.md

```diff
@@ -54,17 +54,94 @@ e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 </box>
 
 --------------------------------------------------------------------------------------------------------------------
-### Viewing help : `help`
+### Adding a guest: `guest add`
+Adds a guest to WedLog.
 
-Shows a message explaining how to access the help page.
+```text
+guest add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [r/RSVP_STATUS] [d/DIETARY REQUIREMENTS] [t/TAG...]
+```
+A guest must have the following tags: `n/NAME`
+
+The following tags are optional: `p/PHONE_NUMBER e/EMAIL a/ADDRESS r/RSVP_STATUS d/DIETARY_REQUIREMENTS t/TAG...`
+
+Acceptable values for `n/NAME`:
+- Alphanumeric word with or without spaces
```

Comment:
I see that you use both the terms 'alphanumeric word' and 'word' -- the meaning of the latter specifically can be quite ambiguous. Would you consider adding a Glossary that defines these terms?

Reply:
to add on on this comment, i think words can have optional plural since names can be more than 1 word. 
e.g. john doe lim

We could maybe use this notation in the UG: alphanumeric word(s).

Reply:
Perhaps a one-time definition of `word` in the glossary will make our UG cleaner.